tests/functional/tests/osd/test_osds.py

```diff
@@ -0,0 +1,23 @@
+
+class TestOSDs(object):
+
+    def test_ceph_osd_package_is_installed(self, CephNode, Package):
+        assert Package("ceph-osd").is_installed
+
+    def test_osd_listens_on_6800(self, CephNode, Socket):
+        assert Socket("tcp://%s:6800" % CephNode["address"]).is_listening
```

Comment:
1 OSD = 4 listened ports, 2 on `public_network` and 2 on `cluster_network`. So if we have 2 OSDs, we need to check from 6800 to 6007.